rss/serializers/rss_2_0.py

```diff
@@ -0,0 +1,86 @@
+"""Serializer for RSS 2.0"""
+
+from typing import Tuple, Optional
+from arxiv import status
+from rss.serializers.serializer import Serializer
+from elasticsearch_dsl.response import Response
```

Comment:
This may seem a little pedantic, but if possible I'd prefer not to couple the serializers with elasticsearch. What you might do instead is 

- define a data struct in ``rss/domain.py`` (e.g. a namedtuple, dataclass, etc) that gives the texture of the data you're using from the ES response
- your ES service integration module (``services/index``) would return an instance of these structs instead; i.e. you'd pull what you need from the ES ``Response``, and leave the ``Response`` behind
- your serializer would expect your domain struct instead of an ES ``Response`` object.

The short term advantage is that you don't need to construct objects from a third-party library to write nice tests. The longer term advantage is that we can swap out ``services/index`` module without touching this code.

Reply:
Created ticket 1675 for this work.